react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+class Stopwatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isStopped: true,
+      countingTime: 0,
+      intervalId: null,
+      timesClicked: 0
+    };
+    this.handlePlayClick = this.handlePlayClick.bind(this);
+    this.handlePauseClick = this.handlePauseClick.bind(this);
+    this.handleReset = this.handleReset.bind(this);
+  }
+
+  handlePlayClick() {
+    if (this.state.timesClicked === 0) {
```

Comment:
To avoid the additional state, it might be better to just check if `intervalId` is `null`. Or if `isStopped` is `true`.

Reply:
Oh right. That is actually better. Thank you!